### PR TITLE
Fixing broken emails

### PR DESCRIPTION
**Pull request by geramirez**
Created a remove function for manual_yaml update solution for 18F/foia-hub/issues/216

Now I just need to update the yaml files.

### CODE REVIEW

**Comment by khandelwal** [comment]
I think this solution is a bit weird. That's not quite how I envisioned the manual files working. @konklone Are there better solutions to this one?

**Comment by geramirez** [comment]
@khandelwal Yeah, the solution became weird, because neither the manual updater nor the foia contacts layering script were meant to overwrite the email field, just add new ones. 
I'll hold off updating the yaml files for now.

**Comment by konklone** [comment]
I think it's probably simpler for the manual layer to completely overwrite the emails - means more data in the manual file, but muuuuch simpler code. What do you think, @ramirezg?

**Comment by geramirez** [comment]
@khandelwal @konklone, works for me but this would sorta sever the connection to FOIA.gov online's contacts. Any additionally contacts FOIA.gov puts up for emails in our manual data won't be stored. If we're okay with that, I'll go ahead and make the change.

**Comment by geramirez** [comment]
Just updated the scraper to overwrite emails with manual data when listed.
I'm going to double check that all the emails are now correct and rebase to take out the commits related to the remove function from the logs.

File: contacts/scraper.py [hunk 1]
```diff
@@ -249,7 +249,7 @@ def update_non_departments(agency_data, manual_data):
 
     agency_data = dict(agency_data)
 
-    list_fields = ['emails', 'common_requests', 'keywords']
+    list_fields = ['common_requests', 'keywords']
```
Comment (khandelwal) [inline contacts/scraper.py:252]: Why did this change?

Comment (geramirez) [inline contacts/scraper.py:252]: This change allows the emails to be overwritten by the manual data.

**Comment by konklone** [comment]
I totally understand the burden we're taking on here -- we're going to have to keep all the emails for an office up to date in the manual data for an office, if we have even one email for them. I think it's worth it, if we're going to meaningfully cut the cord from FOIA.gov. Thanks for doing this, @ramirezg.

**Pull request merged**